scripts/qesap/lib/cmds.py

```diff
@@ -372,11 +372,13 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False, pr
             print(' '.join(command['cmd']))
         else:
             ret, out = lib.process_manager.subprocess_run(**command)
+            log.debug('\n>    '.join(out))
```

Comment:
```suggestion
            log.debug('\n>    '.join(out))
```

Not needed for the moment